analyzers/src/SonarAnalyzer.VisualBasic/Trackers/VisualBasicArgumentTracker.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2023 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.Helpers.Trackers;
+
+public class VisualBasicArgumentTracker : ArgumentTracker<SyntaxKind>
+{
+    protected override SyntaxKind[] TrackedSyntaxKinds => [SyntaxKind.SimpleArgument];
+
+    protected override ILanguageFacade<SyntaxKind> Language => VisualBasicFacade.Instance;
+
+    protected override IReadOnlyCollection<SyntaxNode> ArgumentList(SyntaxNode argumentNode) =>
+        argumentNode is ArgumentSyntax { Parent: ArgumentListSyntax { Arguments: { } list } }
+            ? list
+            : null;
```

Comment:
Wouldn't be better to return an empty collection instead of `null`? See at line 37 how this can easily result in a thrown exception.

Reply:
I added a conditional access 2c137a90e92f8777f6b4e032ad55d1b004502920